src/components/Articles.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+// import { useParams } from "react-router-dom";
+import { getArticles} from "./api";
+
+const Articles = () => {
+    const [articleList, setArticleList] = useState([])
+    // const { article_name } = useParams();
+    
+    useEffect(() => {
+        console.log('hello')
+        getArticles().then(({data}) => {
+            console.log(data);
+            setArticleList(data);
+        });
+    
+      }, []);
+
+console.log(articleList)
+
+  return (
+    
+      <ul>
+        {articleList.map((article) => {
+          return (
+            <li key={article.article_id}>
+              <h3>{article.title}</h3>
+
+              {/* <br /> */}
+              <img className="images" src={article.article_img_url}></img>
+              <h4>{article.topic}</h4>
+              <h4 id="date">{article.created_at}</h4>
```

Comment:
The format of this date's a bit tricky for the user to read, if you wanted to improve this you could change the date format, perhaps using a library like like https://day.js.org/en/